Routes/admin/index.js

```diff
@@ -192,6 +193,36 @@ adminRouter.post('/sessions', (req, res) => {
   });
 });
 
+adminRouter.delete('/sessions/:id', (req, res) => {
+  Session.findByIdAndDelete(req.params.id, (err, data) => {
+    if (!data) {
+      return res.status(400).json({
+        message: 'Сеанса не существует'
+      });
+    }
+
+    if (err) {
+      return res.status(500).json({ message: 'Ошибка сервера' });
+    }
+
+    Ticket.remove({ sessionId: req.params.id }, (err, data) => {
+      if (!data) {
```

Comment:
what will remove return as data? what this check is actually doing? why returning 400?